log/log.go

```diff
@@ -17,13 +18,17 @@ type Logger interface {
 	Debug(msg string, fields ...zap.Field)
 }
 
-func init() {
+func Init() {
 	// TODO: allow to configure the logger
-	lg, err := zap.NewDevelopment()
+	cfg := zap.NewDevelopmentConfig()
```

Comment:
This is yet another debug config. We now have a build tag, a flag and env var...

Reply:
I don't think the debug build tag is used anymore (we should remove it).

For the flag and the env var they have different purposes (one is the debugger, the other one is to enable debug logs).

I can make it a flag for the log level if you want

Reply:
Yes, it would be nice to accept both env vars and flags

Reply:
:+1: